### PR TITLE
Feature/add cr episodes 0 54

### LINKED ISSUE #222: [Coder Radio] Episodes 1-54 Missing
I think this was mentioned in a show once regarding why these episodes aren't in Fireside (something about a migration from a different podcast hosting provider).

Looks like the audio files no longer exist on Libsyn (which looks like what was used prior) for these episodes, but the video does.

### PR DESCRIPTION
**Pull request by ChanceM**
Resolves #222 

Add content for episodes and add image (EP 38).
If podcast_file is None try to play direct files.

### CODE REVIEW

File: content/show/coder-radio/0000.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "michael"
+  ],
+  "guests": [],
+  "sponsors": [],
+  "podcast_duration": null,
+  "podcast_file": null,
+  "podcast_bytes": null,
+  "podcast_chapters": null,
+  "podcast_alt_file": "https://traffic.libsyn.com/jnite/thedust-mp3.mp3",
+  "podcast_ogg_file": "https://traffic.libsyn.com/jnite/thedust-ogg.ogg",
```
Comment (reclaimingmytime) [inline content/show/coder-radio/0000.md:29]: This link doesn't point to a file anymore, including anything with `libsyn.com/jnite` (Jupiter@Nite, the old JB show?)

File: content/show/coder-radio/0000.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  "video_file": "https://201206.jb-dl.cdn.scaleengine.net/coderradio/2012/thedust-432p.mp4",
```
Comment (reclaimingmytime) [inline content/show/coder-radio/0000.md:30]: All `scaleengine.net` links from this PR are also not accessible.

**Comment by reclaimingmytime** [comment]
To to make the old episodes accessible, we would have to re-host the old episodes on a different server. This is part of issue #154, along with all the other unmigrated JB shows.

**Comment by ChanceM** [comment]
> To to make the old episodes accessible, we would have to re-host them on a different server. This is part of issue #154, along with all the other unmigrated JB shows.

Yes, my goal here was to mainly add back content to an active show and to keep it as close to the original content as possible. For that matter JB could rip the audio and add them all back to fireside.

File: content/show/coder-radio/0001.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  "jb_url": "/20497/gateways-to-programming-cr-01/index.html",
+  "fireside_url": "/1"
+}
+
+
+### Episode Links
+
+## Show Notes:
+
+#### What is Coder Radio?
```
Comment (CGBassPlayer) [inline content/show/coder-radio/0001.md:43]: For Episodes that have both `Show Notes` and `Episode Links`, one should be removed so we don't have multiple titles for the same content.

**Review by CGBassPlayer — CHANGES_REQUESTED** [review]
First of all. This is really impressive and is greatly appreciated. I am not sure how you got all the early coder episodes, but I love that you did! :slightly_smiling_face:  

I do think that the `Show Notes` title or the `Episode Links` title in the markdown needs be removed. Having both does not look that good.
image

The other issue I saw when testing the page is that the tags title still appears even when there is no tags. I already have code ready to fix this (just a quick conditional in the template)  so no need to worry about that.

Last thing @reclaimingmytime had already pointed out being the the links that are dead. Not sure where their new location is but we should probably ask JB if they still have them hosted somewhere.

**Review by CGBassPlayer — APPROVED** [review]
I think I can live with some of the links being dead for now. I think what we have now will be good enough for the time being and we can address the 404 links when the archive site starts.

**Pull request merged**